src/Tool-Base/PharoShortcuts.class.st

```diff
@@ -249,6 +251,7 @@ PharoShortcuts >> forwardLineShortcut [
 
 { #category : #'keymaps - editor navigation' }
 PharoShortcuts >> forwardWordShortcut [
+	"Equivalent to: ctrl+right on Windows/Linux, option/alt+right on OSX"
```

Comment:
It seems you forgot to add the same comment for the _**backward**WordShortcut_ method.

Reply:
Thanks for catching this. On my phone now but this can be fixed quickly.